h2o-r/tests/testdir_misc/runit_rmsle.R

```diff
@@ -0,0 +1,52 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+######################################################################
+#Define some helper functions to get RMSLE manually
+
+# Compute the squared log error
+#
+# This function computes the elementwise squared log error for a
+# number or a vector
+#
+# @param actual ground truth number or vector
+# @param predicted predicted number or vector
+sle <- function (actual, predicted) (log(1+actual)-log(1+predicted))^2
```

Comment:
I'd prefer the use of log1p(actual) over log(1+actual), but that's just cosmetics.